ashlar-server/ashlar_server/settings.py

```diff
@@ -0,0 +1,103 @@
+"""
+Django settings for ashlar_server project.
+
+Generated by 'django-admin startproject' using Django 1.8.6.
```

Comment:
Django 1.8 is a pinned requirement in [Ashlar's `setup.py`](https://github.com/azavea/ashlar/blob/develop/setup.py#L19), so when the Dockerfile installs Ashlar it gets Django, too. That's definitely not the right way to do it, but since I'm currently imagining that Ashlar is going to move in the direction of being a separate app that just happens to use Django (as opposed to a Django app for developing other Django apps) I didn't bother doing that here. In any case, explicit is better, so I'll use one of the Azavea Django images so that we can control which Django version is getting installed.

Fully agree that we should upgrade to Django 2.0, although I figured that would be work that needs to be done primarily on Ashlar, not this reference implementation. I was assuming that we would get the reference implementation up and running and then prioritize work (like updating Django) from there, but do you think we should try updating Django first?